src/components/v2/V2Create/tabs/FundingTab/FundingTabContent.tsx

```diff
@@ -134,14 +134,16 @@ export default function FundingTabContent({
 
   return (
     <Row gutter={32} style={{ marginBottom: formBottomMargin }}>
-      <Col span={12}>
+      <Col span={10}>
         <Form form={fundingForm} layout="vertical" onFinish={onFundingFormSave}>
-          <Form.Item label={t`How much do you want to raise?`}>
-            <FundingTypeSelect
-              value={fundingType}
-              onChange={onFundingTypeSelect}
-            />
-          </Form.Item>
+          <Form size="large">
```

Comment:
**Issue:**  `antd` forms can be pretty nasty and like to be set out in a specific way - A single parent `<Form>` component, and children `<Form.Item>` components that are direct parents of some kind of `<Input>`. So we'll need to find another way to restyle this without the `<Form>` inside `<Form>`

Reply:
ahh cool, do you have any suggestion on how to make that change happen? if no direct solution now i can just make it back to normal size

Reply:
Can you just pass the `size="large"` parametre into the parent `Form`?

Reply:
I just made the font larger in the select and removed the second <Form> here